canonical-web-and-design/ubuntu.com
# Add new /managed-apps page

## Done

- Created new /managed-apps page
- Created new modal contact us page
- Created js fallback contact-us and thank-you pages
- Added managed apps to the footer, enterprise mega menu (also added the tco calculator)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/managed-apps - see the [copy doc](https://docs.google.com/document/d/17zJ4vTutqVpn0XzUI784BHdJov8cGW1ttDwIjsvcEzE/edit#)
    - http://0.0.0.0:8001/managed-apps#get-in-touch - see the [copy doc](https://docs.google.com/document/d/1N1S1J1oYVAzp2rs5PF-u8re9GE6c8xpyi17FrzVeOgM/edit)
    - http://0.0.0.0:8001/managed-apps#enterprise - see the [copy doc](https://docs.google.com/document/d/1YBdQvLuqEpEQr_QqyycxMhZr4OaLapMlNJyYKCmPJsQ/edit#)
    - http://0.0.0.0:8001/managed-apps/contact-us - see the [copy doc](https://docs.google.com/document/d/1twkl4QVBl8JXEWumQpu0fYzRWG39wLT9xxDeFzqWgdk/edit#heading=h.rrawfatxvaxc)
    - http://0.0.0.0:8001/managed-apps/thank-you - see the [copy doc](https://docs.google.com/document/d/1ciAw6Lia7UzyiBcNIK6YkyNT8Bi24aq4dnZ85863Cgo/edit#)

- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the copy docs and see that the forms work

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2429 and Fixes https://github.com/canonical-web-and-design/web-squad/issues/2430

## Screenshots

![image](https://user-images.githubusercontent.com/441217/77768205-0d31c100-703a-11ea-81a8-e5578ad26b41.png)
